osc_sdk/sdk.py

```diff
@@ -774,6 +816,78 @@ def build_headers(self, target, json_parameters):
         }
         return signed_headers, canonical_headers, headers
 
+
+class HttpsAdapter(requests.adapters.BaseAdapter):
+    """
+    Adapter to use an external SSL engine (thanks to M2Crypto)
+    https://docs.python-requests.org/en/master/_modules/requests/adapters/
+    """
+
+    def __init__(self, ssl_engine_id, ssl_engine_path, ssl_module_path):
+        super(HttpsAdapter, self).__init__()
+        self.ssl_engine_id = ssl_engine_id
+        self.ssl_engine_path = ssl_engine_path
+        self.ssl_module_path = ssl_module_path
+
+    # BaseAdapter method
```

Comment:
? the method is call send

Reply:
Yes, that method is overloaded from BaseAdapter method

Reply:
Comment removed